Adding a wind average display next to the wind min and wind max. Also…

KyleAnthonyShepherd opened the pull request:
… added a wind risk percentage in the hover tooltip
![image](https://user-images.githubusercontent.com/44480662/183886687-d163a734-ab53-4004-a158-6cca49e82794.png)
![image](https://user-images.githubusercontent.com/44480662/183886719-e2d09214-bd90-4c96-8391-cb0b8150400f.png)

WatchTheFort commented:
I think this is a bit much, in terms of overcomplicating how wind is presented in the game

sprunk commented on luaui/Widgets/gui_top_bar.lua:
```diff
@@ -655,7 +668,18 @@ local function updateWind()
+			risk=risk[maxWind]
+		end
+		if risk == nil then
+			if minWind+maxWind >= 0.5 then
```
Where does the 0.5 come from?

KyleAnthonyShepherd replied:
Other places in gui_top_bar.lua use "if minWind+maxWind >= 0.5 then" to special case no-wind maps. So I'm personally not sure why that method is used.

KyleAnthonyShepherd commented:
Removed average wind number from wind UI box, and moved it to the hover tooltip

![image](https://user-images.githubusercontent.com/44480662/184046700-390e07a3-8552-4316-92ef-a1b4d6b9feda.png)

badosu commented:
LGTM! :+1:
